content/en/docs/faq/applications/remove-built-in-apps.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Remove the Built-in Applications in KubeSphere"
+keywords: "KubeSphere, OpenPitrix, Application, App"
+description: "How to remove the built-in applications in KubeSphere"
+linkTitle: "Remove Built-in Apps in KubeSphere"
+Weight: 16910
+---
+
+As an open-source and app-centric container platform, KubeSphere integrates 15 built-in apps in the App Store that is based on [OpenPitrix](https://github.com/openpitrix/openpitrix).
+
+If you want to remove the built-in apps in KubeSphere, you have to use the account `admin` with the default role of `platform-admin` to log in to the web console of KubeSphere, and then follow the steps as below:
+
+1. Click **Platform** in the upper left corner, and then select **App Store Management**.
```

Comment:
1. Click **Platform** in the upper left corner, and then select **App Store Management**.
->
1. Log in to the web console of KubeSphere as `admin`, click **Platform** in the upper left corner, and then select **App Store Management**.